web-ui/src/components/certifications/CertificationBadges.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { Card, CardContent, CardHeader, Tooltip } from '@mui/material';
+
+import './CertificationBadges.css';
+
+const certificationBaseUrl = 'http://localhost:3000/certification';
```

Comment:
Currently this will only work for me because it relies on endpoints that I implemented in a mock server that I run locally. I need to work with a Micronaut developer to get them ported over.